deployment/debian-package-x64/pkg-src/changelog

```diff
@@ -1,3 +1,9 @@
+jellyfin (10.2.1-2) unstable; urgency=medium
+
+  * Handle ffmpeg dep better
+
+ -- Jellyfin Pakaging Team <packaging@jellyfin.org>  Fri, 22 Feb 2019 23:33:36 -0500
```

Comment:
```suggestion
 -- Jellyfin Packaging Team <packaging@jellyfin.org>  Fri, 22 Feb 2019 23:33:36 -0500
```

Reply:
Thanks, I'll be redoing this entry for the actual release anyways and will fix up then.